index.html

```diff
@@ -7,8 +7,8 @@
     <form id="form">
```

Comment:
So when I was talking about that `keypress` event listener on the submit button, I  may have missed why that piece of code seemed to work. What happens when you comment out the `<form>`? A form will always default to submit when the enter key is pressed. You should be handling the `Enter` keypress in your javascript and not letting the form do the work.